django/conf/__init__.py

```diff
@@ -173,6 +178,8 @@ def __getattr__(self, name):
 
     def __setattr__(self, name, value):
         self._deleted.discard(name)
+        if name == 'DEFAULT_CONTENT_TYPE':
```

Comment:
`Settings` warns projects using the setting. `UserSettingsHolder` warns apps that might be testing with it using `override_settings` and the like.